shared/src/main/scala/mouse/option.scala

```diff
@@ -14,6 +16,8 @@ final class OptionOps[A](private val oa: Option[A]) extends AnyVal {
 
   @inline def toTryMsg(msg: => String): Try[A] = toTry(new RuntimeException(msg))
 
+  @inline def whenExistsA[F[_]: Applicative](f: A => F[Unit]): F[Unit] = cata(f, Applicative[F].unit)
```

Comment:
It's possible to use `traverse_` from `Foldable` with the same semantic:
```scala
Option(42).traverse_(List(_)) // List(())
```

Reply:
ah yeah, good point